numpy/core/src/umath/loops.c.src

```diff
@@ -1448,7 +1451,10 @@ NPY_NO_EXPORT void
 /**begin repeat2
  * #ISA  = , _avx512_skx#
  * #isa  = simd, avx512_skx#
+ * #CHK  = 1, defined(HAVE_ATTRIBUTE_TARGET_AVX512_SKX)#
```

Comment:
This change looks correct. One thing I did notice is that the line above is `#isa  = simd, avx512_skx#, which is a little inconsistent with the AVX2 equivalent higher up: `#isa = , _avx2#`. It looks like the generated code works fine though and is consistent, because the `ISA` template parameter for ` * #ISA = , AVX2#` isn't used at all (looks like that line could be deleted as an additional minor cleanup, but fine to leave that alone too).